qa/tasks/cephfs/test_failover.py

```diff
@@ -306,27 +306,22 @@ def test_simple(self):
         in thrashing tests.
         """
 
-        # Need all my standbys up as well as the active daemons
-        self.wait_for_daemon_start()
+        grace = float(self.fs.get_config("mds_beacon_grace", service_type="mon"))
 
         (original_active, ) = self.fs.get_active_names()
         original_standbys = self.mds_cluster.get_standby_daemons()
 
         # Kill the rank 0 daemon's physical process
         self.fs.mds_stop(original_active)
 
-        grace = float(self.fs.get_config("mds_beacon_grace", service_type="mon"))
-
         # Wait until the monitor promotes his replacement
         def promoted():
-            active = self.fs.get_active_names()
-            return active and active[0] in original_standbys
+            ranks = self.fs.get_ranks()
+            return len(ranks) > 0 and ranks[0]['name'] in original_standbys
```

Comment:
https://pulpito.ceph.com/pdonnell-2021-10-05_13:15:42-fs-wip-pdonnell-testing-20211002.163337-distro-basic-smithi/6423185/

Reply:
Fixed in follow-up commit.